Dockerfile

```diff
@@ -0,0 +1,51 @@
+# -------------------
+# The build container
+# -------------------
+FROM debian:bullseye-slim AS build
+
+RUN apt-get update && \
+  apt-get -y --no-install-recommends install \
+    build-essential \
+    clang \
+    cmake \
+    libcurl4-openssl-dev \
+    libfftw3-dev \
+    libusb-1.0-0-dev \
+    pkg-config \
+    unzip && \
+  rm -rf /var/lib/apt/lists/*
+
+ADD https://github.com/steve-m/librtlsdr/archive/master.zip /root/librtlsdr-master.zip
+RUN unzip /root/librtlsdr-master.zip -d /root && \
+  rm /root/librtlsdr-master.zip && \
+  cd /root/librtlsdr-master && \
+  mkdir -p /root/librtlsdr-master/build && \
+  cd /root/librtlsdr-master/build && \
+  cmake -Wno-dev ../ && \
```

Comment:
Options of this `cmake` are different that what I documented in the README. And I guess it's on purpose, but I will love to learn why you put these ones specifically (it's always good to learn). 

I use `cmake -DCMAKE_INSTALL_PREFIX:PATH=/usr -DDETACH_KERNEL_DRIVER=ON -Wno-dev ..` because I watched the AUR way to compile rtl-sdr on Archlinux, and it worked for me, but I didn't spent too much time to test everything... :/

Anyway, if you have more information about this for me, I will take it :)

Reply:
I pretty much just went for the defaults with the exception of `-Wno-dev`, which suppress developer warnings during the build of `librtlsdr` to minimise noise.

On a Debian install there should be no need to set the install prefix, and `DETACH_KERNEL_DRIVER` I believe just detaches the legacy RTL-SDR Digital TV kernel drivers, which considering we are inside an unprivileged container isn't going to be possible.

This reminded me though that I should add something to the readme about doing this manually on the host system. I'll open a PR for this.